docs/publishing.md

```diff
@@ -11,45 +11,66 @@ Any project owner (or release designee) can complete the following steps to publ
 The steps in this process assume:
 
 - `NPM_TOKEN` is set in the repository secrets. If the token is invalidated or unset, a member of the `@mdn` organization on npm must [create a new token](https://docs.npmjs.com/creating-and-viewing-authentication-tokens) and [add it to the repository's secrets](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository).
-- Your `origin` remote points to `mdn/browser-compat-data`. You may have to adjust the procedure to use a different remote name.
 
-## Prepare the release
+## Prepare for an upcoming release
 
-To create a new version of the package:
+Anticipate a release by keeping a running draft pull request to prepare documentation and metadata for the release. See [#9466](https://github.com/mdn/browser-compat-data/pull/9466) as an example.
 
-1. Figure out the new version number by looking at [past releases](https://github.com/mdn/browser-compat-data/releases). If the release is a non-breaking and data-only update, we're using patch versions. Lets assume that's the case and the next version should be `1.0.3`.
+Start preparing a release when:
 
-2. On your updated and clean `main` branch, run `npm version patch -m "Patch release containing data or non-breaking updates only"`. Locally, this updates `package.json`, creates a new commit, and creates a new release tag (see also the docs for [npm version](https://docs.npmjs.com/cli/version)).
+- the first pull request labeled [needs-release-note 📰](https://github.com/mdn/browser-compat-data/pulls?q=is%3Apr+label%3A%22needs-release-note+%3Anewspaper%3A%22+) has been merged after the most-recent release
+- you're ready to issue a new release
 
-3. Push the commit to `main`: `git push origin main`.
+whichever comes first first.
 
-4. Check if the commit passes our [GitHub Actions workflows](https://github.com/mdn/browser-compat-data/actions).
+1. Start a new branch for the upcoming release. For example, run `git switch -c release-YYYY-MM-DD`, where `YYYY-MM-DD` is the target release date.
 
-5. If the commit passed, push the git tag as well: `git push origin v1.0.3`.
```

Comment:
Where in the new process is the git tag created?

Reply:
Oh I suppose this is a bit surprising. It happens in the GitHub UI, when you publish the release (step 2, in _Publish to npm_). I add a little bit of text to make this bit more explicit.

Reply:
Interesting! :) I've never released BCD with GitHub Actions, so this was a nice "today I learned". Thanks for the update!